.github/workflows/backport.yml

```
@@ -0,0 +1,20 @@
+name: Backport
+on:
+  pull_request:
+    types: [closed]
+jobs:
+  backport:
+    name: Create backport PRs
+    if: github.event.pull_request.merged == true
```

So it doesn't run in PRs made against peoples forks, we have this set in the other actions.

```suggestion
    if: github.repository_owner == 'NixOS' && github.event.pull_request.merged == true
```